exchangelib/services.py

```diff
@@ -1595,14 +1595,14 @@ def call(self, timeout_minutes):
                                      headers=headers)
 
     def get_payload(self, subscription_id, timeout_minutes):
-        get_streaming_events = create_element('{%s}GetStreamingEvents' % MNS)
-        subscription_ids_elem = create_element('{%s}SubscriptionIds' % MNS)
-        subscription_id_elem = create_element('{%s}SubscriptionId' % TNS)
+        get_streaming_events = create_element('m:GetStreamingEvents')
+        subscription_ids_elem = create_element('m:SubscriptionIds')
+        subscription_id_elem = create_element('t:SubscriptionId')
```

Comment:
This is technically fine, but why do we want to hardcode these?

Reply:
if you're asking why we hardcode the `m` and `t`: That's how the create_element function works now. 🤷‍♀️ 
https://github.com/ecederstrand/exchangelib/commit/304840105aae0b3496ab2c600544cdc0e617a934

if you're asking why we hardcode the create_element calls: That's how we originally implemented this. 🤷‍♀️ https://github.com/nylas/exchangelib/commit/e9a6ed6a671142bc9d5e76a0592f7cccd1898768#diff-ac52c34b5cd482fb648c0fdab773fa99

Reply:
Oh, I thought `MNS` was the same as `m:` and was very confused about what you meant by "That's how the create_element function works now". I was like yes, but these are equivalent. Turns out they're not! Nevermind~